config/routes.rb

```diff
@@ -42,6 +42,10 @@
     "/cases/:prosecutionCaseId" \
     "/defendant/:defendantId" \
     "/offences/:offenceId" => "laa_references#create", as: :laa_reference
+  post "/prosecutionCases/laaReference" \
+    "/application/:applicationId" \
```

Comment:
application or applications?

Reply:
Thanks for querying! The ticket and the CC Appeal miro board say "application", but the document we got from HMCTS says "applications" and that seems more likely given the existing endpoint is "cases" not "case". I'm double-checking with Ganga, but I think you're probably right, it should be "applications".

Reply:
Yup, Ganga confirmed it should be "applications". I have updated.